citadel/install_ubuntu_src.md

```diff
@@ -144,7 +144,8 @@ sudo apt-get install -y \
   ruby-dev \
   ruby-ronn \
   swig \
-  uuid-dev
+  uuid-dev \
+  build-essential
```

Comment:
Could you move it up so the list in is alphabetical order? It makes it easier to look for items later.